dblookupext/esdtSupply/logsGetter.go

```diff
@@ -87,16 +91,21 @@ func (lg *logsGetter) getTxLog(txHash []byte) (data.LogHandler, bool, error) {
 	return logFromDB, true, nil
 }
 
-func mergeLogsMap(m1, m2 map[string]data.LogHandler) map[string]data.LogHandler {
-	finalMap := make(map[string]data.LogHandler, len(m1)+len(m2))
+func mergeLogsSlices(m1, m2 []indexer.LogData) []indexer.LogData {
+	logsMap := make(map[string]indexer.LogData, len(m1)+len(m2))
 
-	for key, value := range m1 {
-		finalMap[key] = value
+	for _, value := range m1 {
+		logsMap[value.TxHash] = value
 	}
 
-	for key, value := range m2 {
-		finalMap[key] = value
+	for _, value := range m2 {
+		logsMap[value.TxHash] = value
```

Comment:
this is not a merge but rather a rewrite. Are we sure we want this as it is?

Reply:
removed